configs/rabbitmq/definitions.json

```diff
@@ -0,0 +1,101 @@
+{
+    "users": [
+        {
+            "name": "admin",
+            "password": "admin",
+            "tags": "administrator"
+        },
+        {
+            "name": "service",
+            "password": "service",
+            "tags": ""
+        },
+        {
+            "name": "plugin",
+            "password": "plugin",
+            "tags": ""
+        }
+    ],
+    "vhosts": [
+        {
+            "name": "/"
+        }
+    ],
+    "permissions": [
```

Comment:
just double checking that all these users have the exact same permissions?

Reply:
Yes, that's true here. I can update these to be more strict but that should not affect any of the behavior.

Reply:
all good just checking.